sdk/iot/Azure.Iot.Hub.Service/api/Azure.Iot.Hub.Service.netstandard2.0.cs

```diff
@@ -2,7 +2,7 @@ namespace Azure.Iot.Hub.Service
 {
     public enum BulkIfMatchPrecondition
```

Comment:
`BulkIfMatchPrecondition` and `IfMatchPrecondition` have the same values for the enum now, can we merge them?

Reply:
I am reverting the bulkIfMatch preconditions back to Unconditional.